docs/providers/Serviceplatformen.md

```diff
@@ -0,0 +1,18 @@
+## Serviceplatformen provider
```

Comment:
Suggest we link to https://www.serviceplatformen.dk/

Reply:
done